PointOfDispensingApp/config.js

```diff
@@ -4,5 +4,16 @@ export default {
         UNDETERMINED: 0,
         VACCINATION_PROCEED_YES: 1,
         VACCINATION_PROCEED_NO: 2   
-    }	
+    },	
```

Comment:
Did you delete this file in MVS-521 enabling stuff based on workflow state? If so, these below comments should just carry over to the new place this stuff lives in.

Reply:
I did delete the config.js file in MVS-521; I moved the (now) activeWorkflowStateEnum to store.js